<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>641</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="chessLayout"/>
  </widget>
  <widget class="QLabel" name="page">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>675</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>91</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="numberRowLayout"/>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>641</width>
     <height>70</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="numberColLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
